### ISSUE TITLE
a problem about VAE's  .decoderLayerSizes 

### ISSUE THREAD

**Issue by cqiaoYc** [deeplearning4j/deeplearning4j#3905]
#### Issue Description
I set ".decoderLayerSizes(11,22 , 33,44)", It shows decode_0 layerSize=44, decode_1 layerSize=33, decode_2 layerSize=22, decode_3 layerSize=11 in model page of trainingUI .
I change  .decoderLayerSizes(11,22 , 33,44) to .decoderLayerSizes(44,33 , 22,11), and what shows in trainingUI does't change.

.layer(0, new VariationalAutoencoder.Builder()
                        .activation(Activation.LEAKYRELU)
                        .encoderLayerSizes(44,33 , 22,11) 
                        .decoderLayerSizes(11,22 , 33,44)
                        .pzxActivationFunction(Activation.IDENTITY) //p(z|data) activation function                      
                        .reconstructionDistribution(new GaussianReconstructionDistribution(Activation.TANH))                        
                        .nIn(44)
                        .nOut(5) //Size of the latent variable space: p(z|x) - 32 values
                        .build())                
                .layer(1, new OutputLayer.Builder().lossFunction(new LossMCXENT(getClassWeights())).weightInit(WeightInit.XAVIER).activation(Activation.SOFTMAX).nIn(5).nOut(5).build())
                .pretrain(true)
                .backprop(true)
                .build();
#### Version Information
* Deeplearning4j version: 0.9.1
* platform information (OS, etc) : win10 64
 <dependency>
            <groupId>org.jcuda</groupId>
            <artifactId>jcuda</artifactId>
            <version>0.8.0</version>
        </dependency>
        <dependency>
            <groupId>org.jcuda</groupId>
            <artifactId>jcublas</artifactId>
            <version>0.8.0</version>
        </dependency>

**Comment by cqiaoYc** [comment]
![dl4jtraining ui](https://user-images.githubusercontent.com/29296200/29554382-763420c8-8751-11e7-9da8-9109adedac8b.png)

**Comment by cqiaoYc** [comment]
I don't know whether the display is inconsistent with the actual value in net, or whether the actual value is inconsistent with the settings. I want that the net structure is : 44,33,22,11,Z,11,22,33,44。

**Comment by AlexDBlack** [comment]
Most likely: it's just an issue with the display.

**Comment by cqiaoYc** [comment]
@AlexDBlack You are right, I tracked running, the decode layer sizes were 11,22,33,44 in net.

**Comment by AlexDBlack** [comment]
Fixed here: https://github.com/deeplearning4j/deeplearning4j/issues/3905
